packages-private/vapor-e2e-test/__tests__/e2eUtils.ts

```diff
@@ -0,0 +1,21 @@
+import { type Locator, page, userEvent } from 'vitest/browser'
+
+export const css = (css: string) => page.getByCSS(css)
+export const E2E_TIMEOUT: number = 30 * 1000
+
+export async function enterValue(locator: Locator, text: string) {
+  await locator.fill(text)
+  await userEvent.type(locator, '{enter}')
+}
+
+export function nextFrame() {
+  // this page is not same as Playwright's page
+  // how to wait for the next frame?
+  return page.evaluate(() => {
```

Comment:
@sheremet-va 
I merged the latest code of minor into the current branch, but I encountered a problem and don't know how to solve it. this page is not same as Playwright's page. how to wait for the next frame? I couldn't find a solution in the vitest documentation.